cmd/mungedocs/unversioned_warning.go

```diff
@@ -37,13 +40,14 @@ const unversionedWarningPre = `
 <h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
 
 If you are using a released version of Kubernetes, you should
-refer to the docs that go with that version.
+refer to the docs that go with that version.`
 
+const unversionedWarningLink = `
+
+<!-- TAG RELEASE_LINK, added by the munger automatically -->
```

Comment:
I didn't add the `<!--BEGIN/END MUNGE RELEASE_LINK-->` tags because some of our scripts check `<!--END MUNGE` without further checking the tag contents, so those scripts get confused if I add the tag.

I think the tag at this line is clear enough and does't lose any functionality.